.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -4,8 +4,8 @@ about: Suggest an idea for this project
 
 ---
 
-**Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+**What is your feature request? Please describe.**
+A clear and concise description of the requested feature and how this will improve the user experience. Ex. I'm always frustrated when [...]

```

Comment:
This is a little confusing to me - who is the user?

Maybe reword to something along the lines of: "A clear and concise description of the feature and the benefits it would have."